src/pages/host/getHostEventString.tsx

```diff
@@ -0,0 +1,285 @@
+import React from "react";
+import { HostEventLogData } from "gql/generated/types";
+import styled from "@emotion/styled/macro";
+import { stringifyNanoseconds } from "utils/string";
+import Code from "@leafygreen-ui/code";
+import { StyledLink } from "components/styles";
+import { getUiUrl } from "utils/getEnvironmentVariables";
+import { Collapse } from "antd";
+const { Panel } = Collapse;
+
+const getTerminationString = (monitorOp: string) => {
+  switch (monitorOp) {
+    case `decommissioned`:
+      return `host was decommissioned.`;
+    case `idle`:
+      return `host was idle.`;
+    case `excess`:
+      return `host was decommissioned.`;
+    case `provision_timeout`:
+      return `host took too long for provisioning to complete.`;
+    case `provision_failed`:
+      return `provisioning failed.`;
+    case `expired`:
+      return `expiration time passed.`;
+    default:
+      return `${monitorOp}`;
+  }
+};
+
+export const getHostEventString = (
+  eventType: string,
+  data: HostEventLogData
+) => {
+  switch (eventType) {
+    case `HOST_CREATED`:
+      return `Host created`;
+    case `HOST_AGENT_DEPLOY_FAILED`:
+      return `New agent deploy failed`;
+    case `HOST_PROVISION_ERROR`:
+      return `Host encountered error during provisioning`;
+    case `HOST_STARTED`:
+      return `Host start attempt ${data.successful ? "succeeded" : ""}`;
+    case `HOST_STOPPED`:
+      return `Host stop attempt ${data.successful ? "succeeded" : ""}`;
+    case `HOST_MODIFIED`:
+      return `Host modify attempt ${data.successful ? "succeeded" : ""}`;
+    case `HOST_AGENT_DEPLOYED`:
+      return (
+        <div>
+          {" "}
+          Agent deployed {data.agentRevision ? "with revision" : ""}{" "}
```

Comment:
You can make these strings a bit more succinct if you do "with revision ${data.agentRevision}" as a single template string

Reply:
eslint doesn't allow it :(